src/main/java/Bundle_pl.properties

```diff
@@ -2275,12 +2275,12 @@ passwdVal.passwdReq.lowercase=Ma\u0142e litery
 passwdVal.passwdReq.letter=Litera
 passwdVal.passwdReq.numeral=Znak liczbowy
 passwdVal.passwdReq.special=Znak specjalny
-dataretrieverAPI.noMsgResultsFound=Przepraszamy, nie znaleziono \u017Cadnych wynik�w.
+dataretrieverAPI.noMsgResultsFound=Przepraszamy, nie znaleziono \u017Cadnych wynik�w.
 
 #xlsxfilereader.java
 xlsxfilereader.ioexception.parse=Nie uda\u0142o si\u0119 przeprowadzi\u0107 analizy arkuszy Excel/XLSX ,{0}
-xlsxfilereader.ioexception.norows=Nie znaleziono \u017Cadnych rz\u0119d�w danych w pliku Excel (XLSX)
-xlsxfilereader.ioexception.onlyonerow=Wykryto tylko jeden rz\u0105d danych (nag\u0142�wek kolumny?) w pliku Excel (XLSX).
+xlsxfilereader.ioexception.norows=Nie znaleziono \u017Cadnych rz\u0119d�w danych w pliku Excel (XLSX)
+xlsxfilereader.ioexception.onlyonerow=Wykryto tylko jeden rz\u0105d danych (nag\u0142�wek kolumny?) w pliku Excel (XLSX).
```

Comment:
What's up with this change? Probably we should agree on some kind of unification. Otherwise we always will change labels file in code

Reply:
Hmm it's probably Intellij auto correct, maybe I should turn it off.